MinPlatformPkg/Test/Library/TestPointCheckLib/TestPointImageDump.h

```diff
@@ -0,0 +1,31 @@
+/** @file
+
+Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+Copyright (c) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
```

Comment:
This header file needs an include guard.

Reply:
You're right.  Fixed.